examples/pxScene2d/src/rcvrcore/Logger.js

```diff
@@ -53,8 +76,4 @@ Logger.prototype.message = function(levelNum, message) {
   console.log(this.fullMessage('MESSAGE:'+levelNum, message));
```

Comment:
Let's encapsulate some of the details here.  Can you please have `Logger.prototype.message = function(levelNum, message)` take in either a string or a number?  If it's string then let's convert to a number using `rtl`.